docs/docs/content-scope/content-website.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Content website
+sidebar_position: 1
+---
+
+In a content website, the content scope can be used to separate multiple areas of content, such as website (domain) or language. Each area stores its content independent of the others. Not all modules might use the same scope, for example, the DAM might not be scoped at all. Most of Comet's scope features will work out-of-the-box for this use case.
+
+
+### API: Database
+
+Usually the scope is stored in a "scope" object as an [embeddable](https://mikro-orm.io/docs/embeddables), which the API Generator will then use to create a standardized API that requires passing the scope.
+
+```ts title="api/src/news/entities/news.entity.ts"
+@Embeddable()
+@ObjectType()
+@InputType("NewsContentScopeInput")
+export class NewsContentScope {
+    @Property({ columnType: "text" })
+    @Field()
+    @IsString()
+    domain: string;
+
+    @Property({ columnType: "text" })
+    @Field()
+    @IsString()
+    language: string;
+}
+
+@Entity()
+export class News extends BaseEntity<News, "id"> {
+    @Embedded(() => NewsContentScope)
+    @Field(() => NewsContentScope)
+    scope: NewsContentScope;
+}
+```
+
+### API: GraphQL API
+
+The GraphQL API will have a scope argument (where it makes sense), as shown in the example above.
```

Comment:
I'm not sure what you mean with "the example" above. Do you mean the classes? Shouldn't we add an example here of an actual GraphQL query, e.g., `products(scope: { ... }, filter, sort ...)`?

Reply:
you are right, this is not directly visible in the example above, but would be in the api generated with grud generator for the class above.